### PR TITLE
Updated staking support for CLI

### PR DESCRIPTION
**Pull request by coolcottontail**
1) Updated command line options based on the latest staking message format (Harmony version 110619)
2) Fixed a bug in staking max rate
3) Updated Ledger driver for staking txn to support multiple packets per Txn

### CODE REVIEW

File: pkg/ledger/nano_S_driver.go [hunk 1]
```diff
@@ -219,12 +221,24 @@ func (n *NanoS) SignTxn(txn []byte) (sig [65]byte, err error) {
...
+			p1 = p1First
+		}
+
+		if buf.Len() < 255 {
+			p2  = p2Finish
+		}
+
+		resp, err = n.Exchange(cmdSignStaking, p1, p2, buf.Next(255))
+		if err != nil {
+			return [65]byte{}, err
```
Comment (fxfactorial) [inline pkg/ledger/nano_S_driver.go]: could you please make the 65 be a named constant

File: pkg/ledger/nano_S_driver.go [hunk 2]
```diff
@@ -219,12 +221,24 @@ func (n *NanoS) SignTxn(txn []byte) (sig [65]byte, err error) {
...
+		var p2 byte = p2SignHash
+		if resp == nil {
+			p1 = p1First
+		}
+
+		if buf.Len() < 255 {
+			p2  = p2Finish
+		}
+
+		resp, err = n.Exchange(cmdSignStaking, p1, p2, buf.Next(255))
```
Comment (fxfactorial) [inline pkg/ledger/nano_S_driver.go]: same, 255 given a const name with meaning

File: pkg/ledger/nano_S_driver.go (hunk 1 above)
Comment (coolcottontail) [reply pkg/ledger/nano_S_driver.go]: done

File: pkg/ledger/nano_S_driver.go (hunk 2 above)
Comment (coolcottontail) [reply pkg/ledger/nano_S_driver.go]: done

**Pull request merged**